util/test_python.bash

```diff
@@ -37,7 +37,7 @@ log_info "Running pych --check"
 pych --check
 
 # Copying pyChapel libchpl dependencies
-cp $CHPL_HOME/lib/linux64*/* $VIRTUAL_ENV/share/pych/lib/
+find $CHPL_HOME/lib/ -depth -type f -exec bash -c 'cp $0  $VIRTUAL_ENV/share/pych/lib/' {} \;
```

Comment:
Yeah, but that's no different than what it was doing before.  I agree that it is bad, but I didn't want to dive unnecessarily far down the rabbit hole right now.  I can, though, if you think I should today.